utils/forest_utils/src/cid/mod.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use cid::{
+    multihash::{Code::Blake2b256, MultihashDigest},
+    Cid,
+};
+
+/// Extension methods for constructing `dag-cbor` [Cid]
+pub trait CidCborExt {
+    /// Default CID builder for Filecoin
```

Comment:
Such a lovely new trait! It would be a shame not to have 100% coverage on it! :)

Reply:
test added